products/steamos.md

```diff
@@ -0,0 +1,35 @@
+---
+title: SteamOS
+category: os
+tags: linux-distribution
+iconSlug: steam
+permalink: /steamos
+eolColumn: false
+releaseColumn: false
+discontinuedColumn: true
+releases:
+
+-   releaseCycle: "3"
+    codename: holo
+    releaseDate: 2022-03-01
+    eol: false
+    discontinued: false
+
+-   releaseCycle: "2"
+    codename: brewmaster
+    releaseDate: 2015-11-01
+    eol: true
```

Comment:
The last repo updates to SteamOS 2 were in June 2019 as per https://steamcommunity.com/groups/steamuniverse/discussions/1/1640915206447555580/

```suggestion
    eol: 2019-06-30
```